nodebuilder/tests/prune_test.go

```diff
@@ -45,6 +45,7 @@ import (
 // spin up 3 pruning FNs, connect
 // spin up 1 LN that syncs historic blobs
 func TestArchivalBlobSync(t *testing.T) {
+	t.Skip("FIXME(chatton): error with fx.Replace")
```

Comment:
wanna maybe add another todo in a comment so they can all be grepped by the same string